android/app/src/bte/res/values/strings.xml

```diff
@@ -3,7 +3,7 @@
   <!-- Shown on app info, app switcher, etc -->
   <string name="app_name" translatable="false">PathCheck BT</string>
   <!-- Shown below icon on launcher only -->
-  <string name="app_name_short" translatable="false">PathCheck BT</string>
+  <string name="app_name_short" translatable="false">Path Check BT</string>
```

Comment:
I believe we're sticking with `PathCheck` as official branding
Can we revert this file?

Reply:
yeah, you're right. this is an old PR though (just unmerged), we fixed it in the newer PRs. This should be reflected in the later PR.